Marlin/src/pins/pinsDebug_list.h

```diff
@@ -461,6 +461,9 @@
 #if defined(MAX6675_SS) &&  MAX6675_SS >= 0
   REPORT_NAME_DIGITAL(MAX6675_SS, __LINE__ )
 #endif
+#if defined(MAX6675_SS2) &&  MAX6675_SS2 >= 0
```

Comment:
You can call it `MAX6675_SS2_PIN` and then use `PIN_EXISTS(MAX6675_SS2)` instead. We can rename `MAX6675_SS` to `MAX6675_SS_PIN` also.